### ISSUE TITLE
Release of the pre-trained model weight?

### ISSUE THREAD

**Issue by AlonzoLeeeooo** [pinakinathc/partially-does-it#1]
Hi @pinakinathc,

Appreciate for your excellent work! Would you please release the pre-trained model weight on SketchyCOCO and the inference code? 

Regards,
Alonzo

**Comment by pinakinathc** [comment]
Dear @AlonzoLeeeooo 

I can only get back to this (i.e., upload pre-trained models) in late November. However, the training and validation code is uploaded at [main.py#L66](https://github.com/pinakinathc/partially-does-it/blob/f53e9a7ba167197ef3b9a4a79833455db4bced03/main.py#L66).

If you download SketchyCOCO from [this url](https://github.com/sysu-imsl/SketchyCOCO#1) and NOT change the folder structure, you should be up and running. Just set the data path at [options.py#L16](https://github.com/pinakinathc/partially-does-it/blob/f53e9a7ba167197ef3b9a4a79833455db4bced03/options.py#L16)

**Comment by AlonzoLeeeooo** [comment]
Okay. Thanks for replying. By the way, what GPU do you use for training? How long does the training take?

**Comment by pinakinathc** [comment]
Dear @AlonzoLeeeooo 

As far as I recall, you can hit the numbers on paper in about 5 hours or so. The GPUs are RTX 3090 Ti. Note, I did try with gradient accumulation (via pytorch lightning) for a bigger batch size (which helps a lot).

**Comment by AlonzoLeeeooo** [comment]
> Dear @AlonzoLeeeooo
> 
> As far as I recall, you can hit the numbers on paper in about 5 hours or so. The GPUs are RTX 3090 Ti. Note, I did try with gradient accumulation (via pytorch lightning) for a bigger batch size (which helps a lot).

Thanks for the information!
